package.json

```diff
@@ -82,14 +83,15 @@
         "axios": "^0.27.2",
         "cors": "^2.8.5",
         "deep-extend": "^0.6.0",
-        "dkg-evm-module": "^4.0.5",
+        "dkg-evm-module": "^4.0.7",
         "dotenv": "^16.0.1",
         "ethers": "^5.7.2",
         "express": "^4.18.1",
         "express-fileupload": "^1.4.0",
         "express-rate-limit": "^6.5.2",
         "fs-extra": "^10.1.0",
         "graphdb": "^2.0.2",
+        "interface-datastore": "^8.2.3",
```

Comment:
Do we need this dependency?

Reply:
I'm not able to do npm install without this dependency - it's connected to new version of libp2p